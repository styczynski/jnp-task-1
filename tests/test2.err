Error in ./tests/students.id, line 6: ab111111
Error in ./tests/students.id, line 7: ab01234
Error in ./tests/students.id, line 8: cab12345
Error in ./tests/students.id, line 10: zz-1000
Error in ./tests/students.id, line 11: zżm-1009
Error in ./tests/students.id, line 12: p
Error in ./tests/students.id, line 13: !!!!!!!!
Error in ./tests/students.id, line 15: #
Error in ./tests/students.id, line 16: ąą555555
Error in ./tests/students.id, line 17: ps00000x
Error in ./tests/students.id, line 18: : : :
Error in cin, line 1: grupa1/zadanie1/
Error in cin, line 2: grupa1/zadanie1/ab111111
Error in cin, line 5: grupa2/zadanie2/gh444444 + ij555555
Error in cin, line 6: pr999999
Error in cin, line 7: cd222222
Error in cin, line 8: cd222222
Error in cin, line 8: cd222222
Error in cin, line 9: cd222222
Error in cin, line 10: cd222222
Error in cin, line 10: pr999999
Error in cin, line 11: pr999999
Error in cin, line 11: pr999999
Error in cin, line 11: pr999999
