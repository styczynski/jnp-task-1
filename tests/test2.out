111111;4;
222222;3;
333333;1;
